src/aiokatcp/sensor.py

```diff
@@ -859,7 +855,8 @@ def _update_aggregate(
         old_reading: Optional[Reading[_U]],
     ) -> None:
         updated_reading = self.update_aggregate(updated_sensor, reading, old_reading)
-        self.set_value(updated_reading.value, updated_reading.status, updated_reading.timestamp)
+        if updated_reading is not None:
```

Comment:
Still need to change the return annotation on `update_aggregate` to be `Optional[Reading[_T]]`, and update its docstring to explain that returning None means not updating the sensor. And I think you also need a test for that case.

Reply:
I thought I'd done that. 🤔